### ISSUE TITLE
Can't connect to TheOldReader

### ISSUE THREAD

**Issue by ogustbiller** [jangernert/FeedReader#897]
I have issue Can't connect to TheOldReader. FeedReader write "No connection to server. Check your internet connection and the server URL"
console output are next:
(feedreader:2): feedreader-WARNING **: 21:48:15.234: Utils.gsettingReadString: f
ailed to read org.gnome.feedreader.oldreader access-token
(feedreader:2): dconf-DEBUG: 21:48:15.758: change_fast
(feedreader:2): dconf-DEBUG: 21:48:15.758: change_notify: /org/gnome/feedreader/
oldreader/username
feedreader-INFO: 21:48:17.370: Plugin Name: TheOldReader
feedreader-INFO: 21:48:17.370: Plugin Version: 0.1
feedreader-INFO: 21:48:17.370: Plugin Website: http://jangernert.github.io/FeedR
eader/
feedreader-INFO: 21:48:17.370: Plugin Dir: /app/lib/feedreader/plugins/

**Comment by brendanlong** [comment]
Is this still a problem? The error looks like The Older Reader's API was down, although I don't see anything about this on their [status page](http://status.theoldreader.com/).

**Comment by ogustbiller** [comment]
> Is this still a problem? The error looks like The Older Reader's API was down, although I don't see anything about this on their [status page](http://status.theoldreader.com/).
Yes, i still can not login to the oldreader. I set username and password, but the statys The ol reader is inactive. How i can get additional debug info?

**Comment by belkinsa** [comment]
They're still having issues with their API. So it might be on their side.

**Comment by mbi** [comment]
The Old Reader's API seems to be okay, and works perfectly well with e.g. Liferea. This is definitely a FeedReader problem.
